.github/pull_request_template.md

```diff
@@ -0,0 +1,74 @@
+<!--
+Thanks for opening a PR! Your contribution is much appreciated.
+In order to make sure your PR is handled as smoothly as possible we request that you follow the checklist sections below.
+Choose the right checklist for the change that you're making:
+-->
+
+## Description
+
+<!--
+Link the issue and give some description of the implementation
+For any issue on Github solved by this PR, please use the closing keywords, e.g. Closes #number or Fixes #number
+-->
+
+## Preview
+
+<!--
+Here you can add a video showcasing the new feature or the before/after comparison if it's a fix
+-->
+
+## Dependency changes
+
+<!--
+If there were changes in the dependencies, please mention them here. Added/Updated/Removed dependencies.
+For added dependencies, it's nice to have a description of why it was needed.
+-->
+
+| Dependency | Version        | State   |
+| ---------- | -------------- | ------- |
+| lib1       | 0.1.1          | Added   |
+| lib2       | 0.0.1 -> 1.0.1 | Updated |
+| lib3       | 2.0.1          | Removed |
+
+- lib1 is responsible for...
+
+## Breaking changes
+
+<!--
+Please mention any breaking changes. It's important to have a complete description of the change for reviewers to validate.
+-->
+
+## How to test?
+
+<!--
+If you consider the change needs specific instructions on how to test, use this section to describe it step-by-step.
+-->
+
+1. Go to the page ...
+2. Click on ...
+
+## Good PR checkboxes
+
+- [ ] Change has been tested
+- [ ] Added/Updated tests
+- [ ] Added/Updated documentation
+- [ ] PR title follows [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
+- [ ] Labels are set
+- [ ] Project is linked
+
+## Good Review checkboxes
+
+<details>
+<summary> ℹ️  Copy the snippet and paste in the review field to fill it</summary>
+
+```markdown
+- [ ] I've tested the changes
+- [ ] I've agreed on the unit tests (soon to come)
+- [ ] I've checked the stories
```

Comment:
As stories is mentioned here, should we also mention it on the `Good PR checkboxes` about updating stories? Or is it part of documentation (in which case maybe we can reword a bit to point it more specifically to stories, wdyt?)